stubs/flake8-bugbear/bugbear.pyi

```diff
@@ -0,0 +1,26 @@
+import argparse
+import ast
+from typing import Any
+
+class BugBearChecker:
+    name: str
+    version: str
+    tree: ast.AST | None
+    filename: str | None
+    lines: list[str] | None
+    max_line_length: int
+    visitor: Any
+    options: argparse.Namespace | None
+    def run(self) -> None: ...
+    @staticmethod
+    def add_options(optmanager: Any) -> None: ...
+    def __init__(
+        self,
+        tree: ast.AST | None,
```

Comment:
All 6 `__init__` arguments need `= ...`, because they have a default value. For example, `tree: ast.AST | None = ...`